### PR TITLE
Add target for Reqwest library for Rust

### PR DESCRIPTION
**Pull request by uWynell**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.0.0",
+  "version": "2.1.0",
```
Comment (erunion) [inline package.json:2]: Shouldn't need to do this in a PR, when they publish a release they'll handle this.

**Review by erunion — CHANGES_REQUESTED** [review]
Can you add the necessary test fixtures for this new target?

**Comment by dimitropoulos** [comment]
@uWynell not sure why this closed, but just wanted to point your attention to https://github.com/Kong/httpsnippet/pull/248 which may be merging soon.  one of the things I hope to do after launch is get a rust target in.  if you have any interest in continuing this work please let me know!

**Comment by uWynell** [comment]
@dimitropoulos Yep, I'd like to implmenet a target for reqwest similarly to other targets in #248. If it's not urgent, I will do it in about a week. Or I can hurry up, if necessary.
P.S. I don't remember myself why I closed this pull :\

**Comment by dimitropoulos** [comment]
cool, lemme know how it goes!  Just merged #248 so you should be good to start on it whenever you feel like or have time :)

**Comment by SKPG-Tech** [comment]
Any plans on continuing this?

**Pull request closed without merging**